rock_paper_scissor.dart

```diff
@@ -0,0 +1,110 @@
+import 'dart:io';
+import 'dart:math';
+//Гра "Камінь, ножиці, папір": Користувач грає проти комп’ютера, який випадково вибирає один з варіантів. Визначається переможець.
+
+void main() {
+  //Set user name
+  addUserName();
+  //game flow
+  gameFlow();
+}
+
+//Main data
+
+enum GameElements { rock, paper, scissor }
+
+late String userName;
```

Comment:
don't use late for now. We will discuss what is it and how to use it correctly in future